src/snyk/base/services/scanModeService.ts

```diff
@@ -0,0 +1,55 @@
+import { EXECUTION_PAUSE_INTERVAL, EXECUTION_THROTTLING_INTERVAL } from '../../common/constants/general';
+import { SNYK_CONTEXT, SNYK_MODE_CODES } from '../../common/constants/views';
+import { IContextService } from '../../common/services/contextService';
+
+export class ScanModeService {
+  private _mode = SNYK_MODE_CODES.AUTO;
```

Comment:
just checking - did we agree this is a product decision to default to auto scan?

Reply:
Yes, this is an old PR which servers default autoscan for both Code and OSS. We'll need to adapt it as per today's discussion and announcement on our daily.